### PR TITLE
Alarm for StaleData

### PR DESCRIPTION
**Pull request by RoumenGeorgiev**

### CODE REVIEW

**Comment by MilosKozak** [comment]
did you test it?

File: app/src/main/java/info/nightscout/androidaps/plugins/Overview/OverviewFragment.java [hunk 1]
```diff
@@ -1802,6 +1812,14 @@ public void onClick(View v) {
                         if (notification.nsAlarm != null) {
                             BroadcastAckAlarm.handleClearAlarm(notification.nsAlarm, MainApp.instance().getApplicationContext(), 60 * 60 * 1000L);
                         }
+                        // Adding current time to snooze if we got staleData
+                        log.debug("Notification text is: "+notification.text);
+                        if(notification.text == MainApp.sResources.getString(R.string.nsalarm_staledata)){
```
Comment (MilosKozak) [inline app/src/main/java/info/nightscout/androidaps/plugins/Overview/OverviewFragment.java]: `.equal` shoul be used

**Comment by MilosKozak** [comment]
snnozing stale data alert from AAPS doesn't snooze NS

**Comment by MilosKozak** [comment]
aps snooze in NS worked

**Comment by MilosKozak** [comment]
alarm is not autosnoozed on new data

**Comment by RoumenGeorgiev** [comment]
Fixed all but snoozing stale data alert in NS from AAPS

**Comment by PoweRGbg** [comment]
Not sure how to proceed with alarm snoozing sent to NS.
Now AAPS sends ACK alarm to NS only if alarms is received from NS `if (notification.nsAlarm != null) {`

**Pull request merged**